### DISCUSSION TITLE
How to configure react turbo repo with javascript

### DISCUSSION THREAD

**Discussion by ShivaSredu** [vercel/turborepo#9029] [Help] [answered]
### Summary

Hi

npx create-turbo@latest when i use this cmnd to create turbo repo in react I am getting the project with typescript configuration but I am react developer with javascript so how do i configure the turbo repo with javascript 

Please help me out from this

**Comment by anthonyshew** [comment, marked as answer]
No changes are needed to use Turborepo with JavaScript. We use TypeScript because its more often what we get asked for, but pure JavaScript doesn't require any extra configuration.
